docs/source/_static/dpf_entry.html

```diff
@@ -0,0 +1,3407 @@
+<p class="p-main" id="des_PackageCPythonOpenSourceDev">If you wish to edit and potentially contribute to the DPF-Core or DPF-Post python
+module, clone the repository and install it using pip with the ``-e`` development flag.</p>
+
+<h2 class="h2-main">Connect to a specific server</h2>
+<p class="p-main" id="des_CPythonConnectServer">
+    User may want to set a server with a specific id and port in order to connect to it from a different machine. Here is the code to proceed: 
+</p>
+<simple-scripts id="script_PackageCPythonInstall" python=" pip install ansys-dpf-core
+ pip install ansys-dpf-post"></apis-scripts><simple-scripts id="script_PackageCPythonOpenSourceDev" python=" git clone https://github.com/pyansys/DPF-Core
+ cd DPF-Core
+ pip install -e .
+
+ git clone https://github.com/pyansys/DPF-Post
+ cd DPF-Post
+ pip install -e ."></apis-scripts><simple-scripts id="script_CPythonConnectServer" python="# start a specified server<br># please note that without attribute, this function <br># will start a server with default id and port <br>from ansys.dpf import core as dpf<br>dpf.start_local_server(ip = my_ip, port = my_port)<br><br># Connect to an existing DPF server<br>from ansys.dpf import core as dpf<br>server = dpf.connect_to_server(ip = my_ip, port = my_port)<br>"></apis-scripts></div></div><div class="main_text" id="howToUseIPython"><h1>How to use the IPython package</h1><div><h2 class="h2-main">ACT Console</h2>
+<h3 class="h3-main">Open the ACT console scripting in Mechanical</h3>
+<p class="p-main" id="des_PackageOpenACT">The ACT console can be opened by clicking on "Automation"/"Scripting" menus in Mechanical.</p>
+<h3 class="h3-main">Import DPF and connect it to the current console</h3>
+<p class="p-main" id="des_PackageImportDPFACT">To import DPF�s capabilities into the ACT console, the user should import mech_dpf to access helpers and import the framework contained in the Ans.DataProcessing module. To access data from the current Mechanical windows, DPF�s helpers should be linked to it through the extAPI.</p>
+
+<h3 class="h3-main">DPF's helpers to access mechanical's data</h3>
+<p class="p-main">Diverse Mechanical�s and DPF's data can be accessed: </p>
+<ul class="ul-main">
+    <li  id="des_PackageACTDoc">this documentation can be generated via:</li>
+    <li id="des_PackageACTDataSources">the result files of Mechanical�s analysis: to collect the DataSources (DPF�s entity containing result file paths) the user can write:</li>
+    <li id="des_PackageACTMeshScoping">a mesh selection in the interface: to collect a mesh Scoping (DPF�s entity representing a list of ids of nodes or elements), once a geometry selection is picked out in the interface, it can be accessed via:</li>
+</ul>
+
+<h2 class="h2-main">Python Result</h2>
+<p class="p-main">The Python Result object allows to evaluate a workflow and to visualize its outputs in Mechanical (contours/graph/mesh...).</p>
```

Comment:
The Python Result object allows you to evaluate a workflow and visualize its outputs in Mechanical (contours, graphs, meshing, and so on).

Reply:
updated in TFS